package.json

```diff
@@ -13,8 +13,7 @@
     "Cerner",
     "Terra",
     "autoprefixer",
-    "stylelint",
-    "eslint"
+    "stylelint"
```

Comment:
Here's the set of keyword from original stylelint. 

    "css",
    "less",
    "sass",
    "scss",
    "sugarss",
    "lint",
    "linter",
    "stylelint"

https://github.com/stylelint/stylelint/blob/master/package.json#L6-L13

Reply:
What you added looks good to me. The `lint` keyword was the one I thought would be most valuable.